### PR TITLE
Pin PBFT liveness tests to 1-3 nightly validator

### PR DESCRIPTION
**Pull request by agunde406**
sawtooth-core is currently working toward 2.0 and may not
be compatibly with this current version of PBFT. For now
the liveness test will continue to be run against 1-3,
the "main" branch for further 1.x develpment.

Signed-off-by: Andrea Gunderson <agunde@bitwise.io>

### CODE REVIEW

**Review by ltseeley — DISMISSED** [review]

File: tests/sawtooth-pbft-test.dockerfile [hunk 1]
```diff
@@ -35,10 +35,10 @@ RUN echo "deb [arch=amd64] http://repo.sawtooth.me/ubuntu/nightly bionic univers
     python3-sawtooth-cli \
     python3-sawtooth-intkey \
     python3-sawtooth-rest-api \
-    python3-sawtooth-settings \
```
Comment (peterschwarz) [inline tests/sawtooth-pbft-test.dockerfile:38]: Is this change related?

Comment (agunde406) [reply tests/sawtooth-pbft-test.dockerfile:38]: Yes. 1-3 images do not include the python version of sawtooth-settings so it need to be replaced with the rust version which is included. I updated the commit to clarify these.

**Review by peterschwarz — APPROVED** [review]

**Review by ltseeley — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]
